var/spack/repos/builtin/packages/neurodamus-hippocampus/package.py

```diff
@@ -12,7 +12,7 @@ class NeurodamusHippocampus(NeurodamusModel):
     git      = "ssh://bbpcode.epfl.ch/sim/models/hippocampus"
 
     version('develop', branch='master', submodules=True, clean=False)
-    version('0.4', tag='0.4', submodules=True, clean=False)
+    version('0.4', tag='0.4-1', submodules=True, clean=False)
```

Comment:
Already? Not 0.4.1?

Reply:
we cant delete 0.4, so that "notation" goes for "use this one instead"

Reply:
Hm, OK. I thought the `-1` would be used for the actual version number for re-deployment. This seems more like a brown-bag/patch release (: